modules/azure/src/main/java/org/testcontainers/containers/CosmosDBEmulatorContainer.java

```diff
@@ -0,0 +1,53 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.DockerImageName;
+
+import java.security.KeyStore;
+
+import static org.testcontainers.containers.KeyStoreBuilder.buildByDownloadingCertificate;
+import static org.testcontainers.utility.DockerImageName.parse;
```

Comment:
sorry for not noticing before, but please avoid importing methods statically

Reply:
ok Im editing 👍